Assets/Scripts/Managers/HouseManager.cs

```diff
@@ -14,7 +14,7 @@ public enum HouseMode { Explore, Decorate }
 	[field: SerializeField] public Camera ExploreCamera { get; private set; }
 	[field: SerializeField] public Camera DecorateCamera { get; private set; }
 
-
+	private AudioManager[] audioManager;
```

Comment:
As previously discussed - no reason for this to be an array since the manager is a singleton

Reply:
removed